robotfm_tests/chatops_slack_check.rst

```diff
@@ -0,0 +1,107 @@
+        Should Contain   ${result.stdout}   Chatops message received
+        Should Contain   ${result.stdout}   ${channel}
+ 
+    Execution success    
+        ${result}=         Run Keyword  Get last post_message execution id for slackcat
+        Sleep  10s
+        Should Contain     ${result.stdout}  slackcat posted 1 message lines to chatopsci
+        Log To Console     SLACKCAT: \nSTDOUT: ${result.stdout} \nSTDERR: ${result.stderr} \nRC ${result.rc}
+        #${result}=         Run Keyword  Get last post_message execution id for slackcat          
+
+
+
+
+    *** Keyword ***
+    Hubot Post
+        [Arguments]    ${channel}
+        ${result}=     Run Process    {  echo  -n;  sleep  5;  st2  action  execute  chatops.post_message  channel\=${channel}
+        ...                           message\='Debug. If you see this you are incredibly lucky but please ignore.'
+        ...                           >\/dev\/null;  echo;  sleep  2;}  |  bin\/hubot  \-\-test
+        ...                           cwd=/opt/stackstorm/chatops/    shell=True
+        [return]       ${result}
+
+    Generate Token
+        ${token}=      Generate Random String  32
+        [return]       ${token}
+    
+    Install Slackcat
+        ${result}=      Run Process  wget  https://github.com/vektorlab/slackcat/releases/download/v1.0/slackcat-1.0-linux-amd64  -O  slackcat
+        Sleep  5s
+        # Log To Console   \nSTDOUT: ${result.stdout} \nSTDERR: ${result.stderr} \nRC ${result.rc}
+        # Should Contain  ${result.stderr}  ‘slackcat’ saved
+        File Should Exist  slackcat 
+
+    Add slack token at ${ST2CHATOPS.ENV}
+        ${result}=       Run Process  sudo  sed  -i  -e  's/#  export  HUBOT_ADAPTER\=slack/export  HUBOT_ADAPTER\=slack/g'
+        ...              ${ST2CHATOPS.ENV}  shell=True
+        ${result}=       Run Process  sudo  sed  -i  -e  's/export  HUBOT_ADAPTER\=shell/export  HUBOT_ADAPTER\=slack/g'
+        ...              ${ST2CHATOPS.ENV}  shell=True
+        ${result}=       Run Process  sudo  sed  -i  -e  's/#  export  HUBOT_SLACK_TOKEN\=xoxb-CHANGE-ME-PLEASE/
+        ...              export  HUBOT_SLACK_TOKEN\=${HUBOT_SLACK_TOKEN}/g'
+        ...              ${ST2CHATOPS.ENV}  shell=True
+    
+     ID Execution List Action
+        [Arguments]      ${action_name}
+        ${result}=       Run Process  st2  execution  list  --action\=${action_name}  -a  id  -n  1  -j
+        @{instance id}   Split String      ${result.stdout}    separator="
+        Log To Console   \nINSTANCE ID: @{instance id}[3]
+        [return]         @{instance id}[3]
+
+    Get last post_message execution id for slackcat
+        ${id}=           Run Keyword    ID Execution List Action    chatops.post_message 
+        ${result}=       Run Process    {  echo  '!st2  get  execution  {id}';}  |  slackcat  --channel\=chatopsci
+        ...              --plain  --stream  shell=True
```

Comment:
Actually the WIP part was at line 56. :) .
Also, added the final check for 
> # Look for "ref : chatops.post_message" in the output. 
in the last test case: _Execution from hubot_